projects/ion/src/lib/icon/svgs/iconsText/onu.ts

```diff
@@ -0,0 +1,6 @@
+export default `
```

Comment:
Check if this path is correct, sizes are kinda weird, icon should be bigger I think.
![image](https://user-images.githubusercontent.com/57760325/214642428-04806f8a-3b6d-40a6-bd18-f4a7729b61f2.png)